scripts/irods/test/test_resource_types.py

```diff
@@ -786,7 +786,7 @@ def test_detached_mode_host_not_in_host_list__issue_4421(self):
             lib.make_file(file1, 100)
 
             # put small file
-            self.admin.assert_icommand("iput -R %s %s" % (resource_name, file1), 'STDERR', 'USER_SOCK_CONNECT_ERR')  # iput
+            self.admin.assert_icommand("iput -R %s %s" % (resource_name, file1), 'STDERR', '|'.join(['-347000 USER_SOCK_CONNECT_TIMEDOUT', '-305000 USER_SOCK_CONNECT_ERR']), use_regex = True)  # iput
```

Comment:
We can make this clearer. Consider the following:
```py
_, err, ec = self.admin.run_icommand(<cmd>)
self.assertNotEqual(ec, 0)
self.assertTrue('-347000 USER_SOCK_CONNECT_TIMEDOUT' in err or '-305000 USER_SOCK_CONNECT_ERR' in err)
```

That's probably easier for everyone to understand. This applies to the other calls too.

Thoughts?

Reply:
Agreed.